components/table/src/table.spec.ts

```diff
@@ -827,3 +857,39 @@ class TestIndicatorApp {
   color: 'error' | 'warning' = 'error';
   active = true;
 }
+
+@Component({
+  selector: 'dt-test-app',
+  template: `
+    <dt-table [dataSource]="dataSource">
+      <ng-container dtColumnDef="host" dtColumnAlign="text">
+        <dt-header-cell *dtHeaderCellDef>Host</dt-header-cell>
+        <dt-cell *dtCellDef="let row">{{ row.host }}</dt-cell>
+      </ng-container>
+
+      <dt-header-row *dtHeaderRowDef="['host']"></dt-header-row>
+      <dt-row *dtRowDef="let row; columns: ['host']"></dt-row>
+      <custom-empty-state dtEmptyState></custom-empty-state>
+    </dt-table>
+  `,
+})
+export class TestCustomEmptyStateApp {
+  dataSource: object[] = [{ host: 'host-1' }];
+}
+
+@Component({
+  selector: 'custom-empty-state',
+  providers: [
+    {
+      provide: DtEmptyState,
```

Comment:
Same here, is there a reason why you need to provide it as the `DtEmptyState`? I think this is not needed

Reply:
This comment is obsolete: https://github.com/dynatrace-oss/barista/pull/549#discussion_r376270378